packages/assets/scss/03-organisms/_key-message.scss

```diff
@@ -196,7 +196,10 @@ $colors: (
         display: block;
         height: fit-content;
         padding: 0px;
-        top: 50%;
+
+        @media ($bp-medium-min) {
+          top: 50%;
```

Comment:
Tested in MF, and looks good on mobile 

Before:
https://mayflower.digital.mass.gov/patternlab/patterns/05-pages-marketing-campaign/05-pages-marketing-campaign.html

After:
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-23552_fix-key-message-top/patterns/05-pages-marketing-campaign/05-pages-marketing-campaign.html



However, for certain device widths, I'm still seeing the issue, e.g. Ipad Mini 2019
<img width="606" alt="Screen Shot 2021-12-06 at 5 20 44 PM" src="https://user-images.githubusercontent.com/5789411/144932396-72dc5dac-74e8-435e-bd17-a5c6cfe0bbec.png">